app/assets/stylesheets/components/ui/_teaser.scss

```diff
@@ -39,6 +39,13 @@
 
 .teaser-box__image {
   @include column(12);
+  max-height: 150px; // This will be removed via js if the content is bigger
+  min-height: 150px; 
+  
+  @include respond-to($mq-s) {
+    max-height: 300px; // This will be removed via js if the content is bigger
```

Comment:
Previous MAS practice here would be to use the JS to change the class of the element and create a class with the CSS change. Might be better to use that in this instance.

Reply:
agree that this would be a nicer solution, if this is a lot of work to change then don't worry

Reply:
This is done because if there's no js adjustment we still want the max-height. So just imagine even if there was no js running, we would still want this style there, and it's not included for the js but rather because that's what it should be.